code/modules/karma/karma_packages.dm

```diff
@@ -0,0 +1,106 @@
+// This entire file makes me with DM had enums
+
+/// Base datum for karma packages
+/datum/karma_package
+	/// DB ID. MUST BE UNIQUE. It should follow CATEGORY_ProductName casing. Should be a respective define in [code/__DEFINES/karma.dm]
+	var/database_id = "ERROR_Unset"
+	/// Friendly name to show to the user
+	var/friendly_name = "Some coder was lazy"
+	/// Cost of karma
+	var/karma_cost = 0
+	/// Is this package refundable?
+	var/refundable = FALSE
+	/// Category to show in under the karma store
+	var/category = "UNSET"
+	/// List of "Meta" packages that arent entries, just organisers
+	var/list/meta_packages = list(/datum/karma_package/job, /datum/karma_package/species)
+
+/datum/karma_package/vv_edit_var(var_name, var_value)
+	return FALSE // fuck off
+
+
+// If you are ever removing something from the karma store, **DO NOT** delete its definition datum.
+// Just set redunable = TRUE on it so people can get their karma refund
+
+// Jobs
+/datum/karma_package/job
+	category = "Karma Jobs"
+
+/datum/karma_package/job/blueshield
+	database_id = KARMAPACKAGE_JOB_BLUESHIELD
```

Comment:
I wouldn't rely on defines to keep these names consistent, but this is minor.

Reply:
Its safer than trusting magic strings